src/components/Filters.jsx

```diff
@@ -65,8 +149,10 @@ export const Filters = ()=>{
           type="range"
           min="0"
           max="5"
+          value={rating}
+          onChange={ratingHandler}
         />
       </section>
     </aside>
-    )
-}
\ No newline at end of file
+  );
+};
```

Comment:
break Filters into more components to shorten this, else looks good!!

Reply:
passing context to all filter components was a bit hectic and hard for me to keep track of. So, merged them into one single file.